src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -105,4 +106,11 @@ public void useOutput(double output, double setpoint) {
   public double getMeasurement() {
     return getAverageSpeed();
   }
+
+  @Override
+  public void periodic() {
+    SmartDashboard.putNumber("shooter/speed", getAverageSpeed());
+    SmartDashboard.putNumber("shooter/setpoint", getSetpoint());
+    SmartDashboard.putNumber("shooter/percentSpeed", getAverageSpeed()/getSetpoint());
```

Comment:
percentSetpoint? Not sure. Not a big idea

Reply:
resolved in f2e6a54